db/schema.rb

```diff
@@ -607,6 +611,19 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "session_blocked_lists", force: :cascade do |t|
+    t.string "devise_session"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "sessions_blacklists", force: :cascade do |t|
+    t.string "sessionid", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["sessionid"], name: "index_sessions_blacklists_on_sessionid"
+  end
+
```

Comment:
@shivashankar-f22labs not sure these tables should be here

Reply:
@osahyoun 
Will update the schema